common/mediasdk_directories.py

```diff
@@ -166,6 +168,20 @@ def get_commit_dir(cls, branch, build_event, commit_id):
 
         return pathlib.Path(cls._builds_root_path) / branch / build_event / commit_id
 
+    @classmethod
+    def get_artifact_url(cls, dir):
```

Comment:
Need to do the same function for closed source.
Build runner will fail "copy" stage without it.